playground/src/index.css

```diff
@@ -18,86 +18,51 @@ html, body, #root {
   height: 100vh;
 }
 
-.graphiql-defradb-plugin {
-  display: flex;
-  flex-direction: column;
-  gap: 16px;
+.swagger-ui .opblock .opblock-summary {
```

Comment:
question: Is any of this css hand crafted, or is it some default stuff copy-pasted from somewhere?

Reply:
I went through and manually updated the styles for dark mode. They don't have css variable support in Swagger UI so this was the next best option.